Makefile.am

```diff
@@ -102,6 +103,11 @@ ppmclibs/Makefile: ppmclibs/Makefile.PL
 $(PERL_MODULE): ppmclibs/Makefile
 	$(MAKE) -C ppmclibs
 
+$(OPENVSWITCH_SERVICE):
+	rm -f $@ $@.tmp
+	sed -e 's,@pkglibexecdir[@],$(pkglibexecdir),g' '$(top_srcdir)/$@.in' > '$(top_srcdir)/$@.tmp'
```

Comment:
that's not really the right way to do it, per [this page](https://www.gnu.org/software/autoconf/manual/autoconf-2.63/html_node/Installation-Directory-Variables.html) - at least as I read it, see the bit from "Most of these variables have values that rely on prefix or exec_prefix." onwards. Note particularly "A corollary is that you should not use these variables except in makefiles".